webauth-certificates.yml

```diff
@@ -0,0 +1,24 @@
+- name: Install certificates for Stepup-Webauthn GSSP
+  hosts: stepup-webauthn
+  become: True
+  tags: stepup-webauthn-certificates
+
+  tasks:
+    - name: Find path for webauthn GSSP
+      find:
+        paths: /opt/www
+        file_type: any
+        recurse: no
+        follow: yes
+        patterns: "webauthn.*"
+      register: dirs_matched
```

Comment:
See #177: Why not use the info from group_vars?

Reply:
Changed to build the path from group_vars